spec/controllers/admin_censor_rule_controller_spec.rb

```diff
@@ -280,18 +290,18 @@ def create_censor_rule
           expect(flash[:notice]).to eq(msg)
         end
 
-        it 'purges the cache for the info request' do
-          info_request = FactoryGirl.create(:info_request)
-          censor_rules = double
-          allow(info_request).to receive(:censor_rules) { censor_rules }
-          allow(InfoRequest).to receive(:find) { info_request }
-          censor_rule = FactoryGirl.build(:info_request_censor_rule, :info_request => info_request)
-          allow(censor_rules).to receive(:build) { censor_rule }
+        it 'calls expire_requests on the new censor_rule' do
+          allow(InfoRequest).to receive(:find) { @info_request }
```

Comment:
(It would also be an opportunity to replace @info_request with a let-ed equivalent)